hudi-spark-datasource/hudi-spark3.5.x/src/test/java/org/apache/hudi/spark3/internal/TestReflectUtil.java

```diff
@@ -42,7 +44,7 @@ public void testDataSourceWriterExtraCommitMetadata() throws Exception {
     InsertIntoStatement newStatment = ReflectUtil.createInsertInto(
         statement.table(),
         statement.partitionSpec(),
-        scala.collection.immutable.List.empty(),
+        ((scala.collection.immutable.Seq<String>) scala.collection.immutable.Seq$.MODULE$.empty()).toSeq(),
```

Comment:
?. I guess disabled because it doesn't work?

Reply:
I plan to remove `ReflectUtil.createInsertInto` in another PR.  `ReflectUtil.createInsertInto` is not used in the codebase (a leftover function used before, but abandoned now).

Reply:
FYI this is done in #11135 .